packages/react/src/FileList/FileList.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { DocumentIcon } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
+import { Button } from '../Button'
+import { Icon } from '../Icon'
+import { formatFileSize } from '../common/formatFileSize'
+import { formatFileType } from '../common/formatFileType'
+
+export type FileListProps = {
+  files: FileList
```

Comment:
Let’s try the subcomponent approach here that we often use, as suggested in [Slack](https://datalab-amsterdam.slack.com/archives/C06B35WFZM2/p1733223833899999).

```
<FileList>
  <FileList.Item />
  <FileList.Item />
</FileList>
```

Reply:
ALso if you do not require a user to use FileList.Item baecause it's a loop in this component??

Reply:
Yes, we do so with various other components. Likewise, we could have done something like `<DescriptionList items={[term, description, ...]} />`. Object props are just not very clear, even though it’s a bit different for File List because we get the object straight from the browser.